Sources/ArcGISToolkit/Components/Offline/PreplannedListItemView.swift

```diff
@@ -21,13 +21,19 @@ struct PreplannedListItemView: View {
     /// The view model for the preplanned map.
     @ObservedObject var model: PreplannedMapModel
     
+    /// The currently selected map.
+    @Binding var selectedMap: Map?
+    
     /// A Boolean value indicating whether the metadata view is presented.
     @State private var metadataViewIsPresented = false
     
-    /// The closure to perform when the map selection changes.
-    let onMapSelectionChanged: (Map) -> Void
-    /// The closure to perform when the map is removed from local disk.
-    let onDeletion: () -> Void
+    /// A Boolean value indicating whether the selected map area is the same
+    /// as the map area from this model.
+    /// The title of a preplanned map area is guaranteed to be unique when it
+    /// is created.
+    var selectedMapFromThisModel: Bool {
```

Comment:
```suggestion
    var isSelected: Bool {
```
This Boolean should be renamed to read as an assertion.